### ISSUE TITLE
How to Split Pane?

### ISSUE THREAD

**Issue by kmaustral** [oxdc/obsidian-vertical-tabs#57]
The Split Pane command puts a copy of the note into a new pane. With Vertical Tabs, this just puts a new copy of the note in the sidebar. How can you put a copy of the note in a new pane with Vertical Tabs enabled?

**Comment by oxdc** [comment]
Hi! You may have enabled the Zen Mode, which would hide inactive splits (panes). Please disable it to see other splits. I recorded a video about how to split the screen and hide & focus with Zen Mode:


https://github.com/user-attachments/assets/5c0299f6-59aa-4514-a764-c76b25df27a4

**Comment by kmaustral** [comment]
Yes, you're right! I'm glad I can keep trying out this promising plugin.
